packages/upset/src/components/ElementView/ElementTable.tsx

```diff
@@ -22,16 +24,29 @@ function useRows(items: Item[]): GridRowsProp {
   }, [items]);
 }
 
+/**
+ * Hook to generate columns for the DataGrid
+ * @param columns columns to be displayed
+ * @returns array of columns with item field to be displayed in the DataGrid and headerName
+ */
 function useColumns(columns: string[]) {
   return useMemo(() => columns.map((col) => ({
     field: col,
+    // Prefixed with _ since that's how the Item object is structured
     headerName: col === '_id' ? 'ID' : col === '_label' ? 'Label' : col,
   })), [columns]);
 }
 
-export const ElementTable: FC<Props> = ({ id }) => {
+/**
+ * Table to display elements
+ */
+export const ElementTable: FC = () => {
+  const currentIntersection = useRecoilValue(currentIntersectionSelector);
+  const currentSelection = useRecoilValue(selectedElementSelector);
   const attributeColumns = useRecoilValue(attributeAtom);
-  const elements = useRecoilValue(elementSelector(id));
+  const elements = currentSelection?.selection
+    ? useRecoilValue(selectedItemsSelector)
+    : useRecoilValue(elementSelector(currentIntersection?.id));
   const rows = useRows(elements);
   const columns = useColumns(['_id', '_label', ...attributeColumns]);
```

Comment:
We can clean this up here
```suggestion
  const columns = useColumns(['_label', ...attributeColumns]);
```

Reply:
Addressed in 0b9328c6ad6a6406dbd04dc6b20aeb495d6014db